src/courses/delta/03.md

```diff
@@ -0,0 +1,87 @@
+---
+order: 03
+next: 04.md
+title: 3. Delta Process CLI Tools
+author: Daniel Medina, George Dias
+---
+
+## 3.1 CLI Tools
+
+The delta process consists of one or both of the following commands:
+
+1. Update Controls for Delta (`update_controls4delta`)
+2. Delta (`delta`)
+
+::: note About InSpec Profiles
```

Comment:
This feels out of place here. I don't yet know enough as a student to differentiate or need to and so this placement just provides some confusion.